### PR TITLE
Update .gitignore, add Rubocop/Standard gems to Gemfile

### PR DESCRIPTION
**Pull request by yulgolem**
Does not affect production code.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -33,6 +33,8 @@
 .rubocop-ruby.yml
 .rubocop.yml
 
+.idea/
```
Comment (artur-beljajev) [inline .gitignore]: Developer-specific files/folders should not be here. How about using local gitignore?

File: Gemfile [hunk 2]
```diff
@@ -33,6 +33,8 @@ end
 group :development do
   gem 'listen', '>= 3.0.5', '< 3.2'
   gem 'pry'
+  gem 'rubocop', '~> 0.75.1', require: false
```
Comment (artur-beljajev) [inline Gemfile]: Why those gems are needed here?

Comment (yulgolem) [reply Gemfile]: That's for local linting in RubyMine in dev-time.

Comment (artur-beljajev) [reply Gemfile]: Doesn't `gem install rubocop` and `rubocop %file%` do the stick?

Comment (yulgolem) [reply Gemfile]: Hmmm... well, ok, I`ll just decline that.

**Pull request closed without merging**